### ISSUE TITLE
Add timestamps bookmark function for subless listening experience similar to some youtube bookmark extensions

### ISSUE THREAD

**Issue by phamthaison1207** [killergerbah/asbplayer#374]
When I am watching anime or anything without subs, I want to be able to save a timestamp of where I couldn't decode what they were saying and noticed there isn't such a feature yet in asbplayer like a timestamp bookmark extension for it.
So I think suggesting this feature will be really useful for any listening based learner like me.
![image](https://github.com/killergerbah/asbplayer/assets/63585568/bb563c13-b944-4b6b-9647-a7efcfebb39b)

**Comment by phamthaison1207** [comment]
ah yeah, for my training, I don't stop and record audio (it spoils the content for my brain) but just mark the timestamp so I have to use listening to comprehend instead of making a habit of relying on the spoilers from asbplayer

**Comment by phamthaison1207** [comment]
like: from a to z: rely purely on listening to decode the content : hit or miss type of training, no stopping and turning on subs
![image](https://github.com/killergerbah/asbplayer/assets/63585568/5537e91e-9358-4b20-81fa-c683c6af910f)

**Comment by killergerbah** [comment]
In that case you could use the 'mine subtitle' keyboard shortcut (not: 'mine subtitle and open anki dialog'). Asbplayer should copy the subtitle into your mining history and continue playing. You will have to try to not look at the notification that pops up however.

**Comment by naiza-asaad** [comment]
Hi @killergerbah, is there a button to save to the mining history? I'm thinking of buying a Windows tablet for portability but I'm not sure how to save the sentence without relying on the keyboard shortcut.

**Comment by killergerbah** [comment]
@naiza-asaad If you want to mine from streaming video, you'll just need to confirm that you can install some version of Chromium that supports chrome extensions onto your tablet. Assuming asbplayer can be installed then you should be able to use the side panel OR video overlay (enabled by default or via 'streaming video' settings) to mine using a button. You can give it a try on desktop to see if that's something that would work for you.

If you want to mine local video files, the + button that appears next to every subtitle in the subtitle list should suffice.

**Comment by naiza-asaad** [comment]
It's just going to be a Windows tablet PC so I guess it'll have a similar look as on my laptop right now? I didn't notice that the button at the side panel also adds to the mining history so this will work for me, thanks! Although currently it always opens the export Anki dialog. Is there/will there also be a button equivalent to Mine current subtitle (without opening the Anki dialog)?

**Comment by naiza-asaad** [comment]
I looked at the settings after typing my reply, and I found that I can actually set the default mining button to None so it just saves the sentence to the history.

**Comment by killergerbah** [comment]
Yeah that's the setting that'll control the button behavior. Hopefully that'll work for you.

**Comment by phamthaison1207** [comment]
> In that case you could use the 'mine subtitle' keyboard shortcut (not: 'mine subtitle and open anki dialog'). Asbplayer should copy the subtitle into your mining history and continue playing. You will have to try to not look at the notification that pops up however.

**Comment by killergerbah** [comment]
The copy button in the video overlay? Where do you want to move it?

**Comment by phamthaison1207** [comment]
After thinking about it for a while, maybe moving it wouldn't solve the problem:
I think not showing the content of what it copied should fix it definitely (for immersion)
